content/applications/finance/accounting/fiscal_localizations/localizations/colombia.rst

```diff
@@ -59,7 +59,15 @@ look for the *Colombian Electronic Invoice* section.
 Using the Testing mode it is possible to connect with a Carvajal T&S
 testing environment. This allows users to test the complete workflow
 and integration with the CEN Financiero portal, which is accessible
-here: https://cenfinancierolab.cen.biz
+here:
+
+Carvajal CTS
```

Comment:
Here it could be:
`Carvajal CTS (Carvajal T&S)`

Reply:
I've left off the first "Carvajal" to avoid saying "Carvajal Carvajal T&S (Carvajal T&S)" ;-)